change/@ni-nimble-components-f7110334-7d62-4d62-8f37-10ba83de8727.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Explicitly added @fredvisser to validate the interactions of this workaround (see PR description for tab stop behavior and link to sl-header storybook)